pkg/segment/writer/packer.go

```diff
@@ -672,6 +680,38 @@ func encJsonNumber(key string, numType SS_IntUintFloatTypes, intVal int64, uintV
 	return valSize
 }
 
+func (ss *SegStore) updateColValueSizeInAllSeenColumns(colName string, size uint32) {
+	currentSize, ok := ss.AllSeenColumns[colName]
+	if !ok {
+		if ss.RecordCount > 0 {
+			// Not the first record in the segment, so the col value size is not consistent.
+			ss.AllSeenColumns[colName] = -1
+			currentSize = -1
+		} else {
+			ss.AllSeenColumns[colName] = 0
```

Comment:
we can just set `ss.AllSeenColumns[colName] = inSize` and return from here

Reply:
Yes, I did this way so that I can check the size. But now since there is no limit. I can just assign the size here and return.